Dockerfile

```diff
@@ -102,6 +104,8 @@ RUN apt-get update && \
 RUN mkdir -p /apk-pre.d /apk.d && \
   curl -L -o /apk-pre.d/FDroid.apk https://f-droid.org/FDroid.apk && \
   curl -L -o /apk-pre.d/firefox.apk https://ftp.mozilla.org/pub/mobile/releases/68.9.0/android-x86_64/en-US/fennec-68.9.0.en-US.android-x86_64.apk && \
+  curl -L -o /apk-pre.d/aurora_store.apk https://auroraoss.com/AuroraStore/Stable/AuroraStore_3.2.9.apk && \
```

Comment:
I'm not familiar with this, but assuming from screenshots in https://gitlab.com/AuroraOSS/AuroraStore , it seems redistributing APKs from Google store without permission?